doc/api/crypto.md

```diff
@@ -656,6 +656,54 @@ assert.strictEqual(aliceSecret.toString('hex'), bobSecret.toString('hex'));
 // OK
 ```
 
+### ECDH.convertKey(key, curve[, inputEncoding][, outputEncoding][, format])
+<!-- YAML
+added: v10.0.0
```

Comment:
This should be `added: REPLACEME` (it might be a different version on other release branches, not necessarily `v10.0.0`).